modules/datacube.py

```diff
@@ -1,6 +1,9 @@
 import xarray as xr
 
+
 def check_keys(actual_keys, required_keys):
+    # tests that required keys are a subset of actual keys
+
```

Comment:
I know this code is not part of your changes, but ... 

Line 13 cannot be reached, and this function has only two possible behaviours: raise KeyError, or return True. Suggest you remove variable `passed` altogether, and replace line 14 with `return True`.